utils/Navigation.js

```diff
@@ -38,11 +38,17 @@ class Navigation {
       .sort((a, b) => a.order - b.order);
     pages.forEach(page => navigation.insertPage({ page }));
 
-    // FIXME remove this block once merchant integrations has a landing page
-    const merchantIntegrations = navigation.findItem({ title: 'Merchant Integrations' });
-    if(merchantIntegrations) {
-      merchantIntegrations.to = merchantIntegrations.children[0].to;
-    }
+    const sectionHeaderTitle = [
```

Comment:
Let's put the FIXME back in here. The intention is to have landing pages for all the sections so this block of code won't be needed once that's implemented.

Reply:
Done

Reply:
It isn't just merchant integrations now though is it? This handles the landing pages for all 3 sections. We should update the comment

Reply:
I have updated the changes. :)